annotation/core/models.py

```diff
@@ -48,10 +61,13 @@ class Generation(models.Model):
     prompt = models.ForeignKey(Prompt, on_delete=models.DO_NOTHING)
     decoding_strategy = models.ForeignKey(DecodingStrategy, on_delete=models.DO_NOTHING)
     body = models.TextField()
-    boundary = models.IntegerField()
+
+    @property
+    def boundary(self):
+        return self.prompt.num_sentences
 
     def __str__(self):
-        return self.body
+        return self.body+1
```

Comment:
What is the point of this +1? Is this to coax some off-by-one error?

Reply:
Bug, I fixed it!